src/app/public/modules/autocomplete/autocomplete-input.directive.ts

```diff
@@ -214,7 +216,12 @@ export class SkyAutocompleteInputDirective implements OnInit, OnDestroy, Control
   public onValidatorChange = () => {};
 
   private setAttributes(element: any): void {
-    this.renderer.setAttribute(element, 'autocomplete', 'off');
+    /**
+     * Need a unique value for autocomplete to be disabled on Chrome.
+     * https://bugs.chromium.org/p/chromium/issues/detail?id=914451
+     */
+    const id = ++uniqueId;
+    this.renderer.setAttribute(element, 'autocomplete', `skyux-autocomplete-${id}-off`);
```

Comment:
If this works for Chrome, does it work for all the other browsers, too? I wouldn't want to introduce a hack for one browser at the cost of the others...

Reply:
@Blackbaud-SteveBrush we've employed a similar approach to disable autocomplete that works across browsers, I'll send you the link to the internal repo.